Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
 gem 'cocoapods', '~> 1.3.1'
-gem "fastlane", "~> 2.64"
-gem "xcov", "~> 1.3.0"
+gem "fastlane", "~> 2.92"
```

Comment:
it seems there is already a 2.93 release, not sure if it is worth it to change it again :)